configure

```diff
@@ -1048,6 +1052,8 @@ for ac_option in $@; do
 	--disable-libunity)       _libunity=no    ;;
 	--enable-opengl)          _opengl=yes     ;;
 	--disable-opengl)         _opengl=no      ;;
+	--enable-dispmanx)	  _dispmanx=yes   ;;
+	--disable-dispmanx)	  _dispmanx=no    ;;
```

Comment:
These two lines are using tabs where they should be using spaces (matching the surrounding code). Hmm, I guess the Code Formatting Conventions in the ScummVM Wiki should mention that (at least I just didn't find it). But as  a rule of thumb, whenever submitting code for some project, it helps to adapt your code changes to "blend in" with the rest. One trick to notice such problems with tabs is to temporarily set your local tab width to 20 or so, then problematic spots stand out. Another is to grep for places where a tab occurs next to a regular space.